src/View/Helper/HtmlHelper.php

```diff
@@ -5,6 +5,12 @@ class HtmlHelper extends \Cake\View\Helper\HtmlHelper
 {
     use OptionsAwareTrait;
 
+    private $grid = [];
+
+    private $gridOptions = [];
+
+    private $gridCounter = 0;
```

Comment:
I would say that the FoC plugins should follow the same standards of core CakePHP where no private methods or properties are allowed.